ui/src/submissions/jobs/schemas/job.js

```diff
@@ -71,14 +71,18 @@ const jobSchema = object().shape({
         .label('Contact email'),
     })
   ),
-  deadline_date: date()
-    .required()
-    .test(
-      'is-valid-deadline-date',
-      'Deadline should be within next year',
-      isValidDeadlineDate
-    )
-    .label('Deadline'),
+  deadline_date: date().when('status', {
+    is: 'closed',
+    then: date(),
```

Comment:
We still need to check that the `date` is a valid one (proper format and not just a random string), no?

Reply:
This checks that it's not a random string but a valid date string. For the format we don't check this at the moment, since `DatePicker` handles it. (we could add it if some users start doing some hacky stuff)